apps/cookbook/src/app/showcase/section-header-showcase/section-header-showcase.component.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@kirbydesign/core/src/scss/utils';
+@import '@kirbydesign/core/src/scss/utils';
```

Comment:
I wonder why this specifically is an import. 
Can we change it to use now that we are at it? 🤔